library/common/types/c_types_utility.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "library/common/types/c_types.h"
+
+namespace Envoy {
+namespace Utility {
+
+/**
+ * Copy envoy_data into an std::string.
+ * @param s the envoy_data to copy.
+ * @return std::string the string constructed from s.
+ */
+std::string convertToString(envoy_data s);
```

Comment:
Please merge main. This functionality has moved to Buffer::Utility. All in concert to help with #1302